cmd/config/config.go

```
@@ -189,6 +195,7 @@ func ReadConfigRun(configDir string, flags *pflag.FlagSet, mounter mount.Interfa
 	err := viper.Unmarshal(cfg, setDecoder, decodeHook)
 	if err != nil {
 		cfg.Logger.Warnf("error unmarshalling RunConfig: %s", err)
+		return cfg, err
```

Im not sure we even need to return an error here, something to discuss for sure. I mean, the config is not strictly necessary, and we are already warning the user that we could not load it....but probably is good UX to rejext continuing if we cant load the user config....

---

Yea, was thinking it's safer to error out if the config is broken instead of potentially running something unwanted..

---

I removed the return in latest commit, we could have a discussion later if we should continue running commands if we fail to load configuration.

---

:+1: Although I think this is the correct approach, we may be kind of breaking the workflow here that we didnt before..